### PR TITLE
7.0 add extended binary field

### PR DESCRIPTION
**Pull request by ntodorova**
New version of binary field, including creation of ir_attachment object.

### CODE REVIEW

**Comment by ntodorova** [comment]
@hbrunn can you please give me some tips to pass the coveralls test? : ))

File: extended_binary_field/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    ],
+    'css': [
+    ],
+    'demo': [
+    ],
+    'test': [
+    ],
+    'active': False,
+    'installable': True,
+    'auto_install': True,
```
Comment (legalsylvain) [inline extended_binary_field/__openerp__.py]: Why is this module auto_installable ?

**Comment by pedrobaeza** [comment]
Is anyone interested in this one yet?

**Comment by ntodorova** [comment]
I would like to bring it to the end if someone can help me with the coveralls. : )

**Comment by hbrunn** [comment]
what kind of help do you need? what we want in the end is 100% test coverage, the good reads are https://www.odoo.com/documentation/8.0/reference/testing.html and https://docs.python.org/2/library/unittest.html

**Comment by ntodorova** [comment]
But I think that the branch coveralls were decrease before my commit, or? And  on the branch in general, or I am wrong?

**Comment by hbrunn** [comment]
yes, you're wrong. hopefully at least. the script is meant to measure how much test coverage a pr adds, and given you add code without adding tests, the pr decreases coverage

**Comment by ntodorova** [comment]
ok, than you!

**Pull request closed without merging**